Makefile

```diff
@@ -40,9 +40,7 @@ install-linter:
 
 install-dev:
 	# a tool to vendor non-go files
-	# installed from a fork that doesn't ignore replace directive
```

Comment:
@dshulyak which versions of Go and gomodvendor have you used? 

I just tested on `$ go version go1.12.5 darwin/amd64` and HEAD of github.com/goware/modvendor and it works fine. I can build a binary and run tests.

Reply:
pretty sure it can't work fine unless author merged a fix for replace directives

Reply:
yes author merged this change https://github.com/goware/modvendor/commit/ce72b408a8fee27135ceec363f76cdac3350010e